ocfweb/docs/docs/services/web/django.md

```diff
@@ -1,6 +1,133 @@
 [[!meta title="Django"]]
 
-**Note: If you are using a group account, you may wish to consider
-[[apphosting|doc services/webapps]] instead. The OCF is currently
-working on updating this documentation to better help users help
-set this up.**
+**Note: These instructions are primarily intended for individual user accounts.
+If you are using a group account, you may wish to consider
+[[apphosting|doc services/webapps]] instead.**
+
+Django is a popular web framework for Python applications. At the OCF, you can
+host it using our regular web hosting, though for groups with virtual hosting,
+we strongly recommend instead using our [[application hosting service|doc
+services/webapps]] instead, as it provides a first-class solution for webapps.
+
+## Running a Django app
+
+The steps below assume you have already created a Django application. If you
+haven't, take a look at [the Django documentation][django-docs] for help with
+that.
+
+1. Copy your Django application to your OCF account. For example, you can use
+   `scp` to upload it, or if you're using source control, clone it using `git`
+   (or another tool) into your home directory.
+
+   In the steps below, we'll assume you put it at `~/app` in your home
+   directory.
+
+2. Inside your project directory, create a virtualenv and install your
+   dependencies. For example, to create a Python 3 virtualenv, try:
+
+   ```bash
+   user@tsunami:~/app$ virtualenv -p python3 venv
```

Comment:
maybe recommend `python3 -m venv myenv`?

Reply:
+1, does not require `pip install virtualenv` in Python 3

Reply:
I would do this, but this requires `python3-venv` be installed, which it isn't currently on tsunami (although we could install it there perhaps). `virtualenv` on the OCF systems comes from the `virtualenv` package, which we do have installed, apparently as a dependency of `tox`, which is installed here: https://github.com/ocf/puppet/blob/1df64fb6b89d33d21c3eed1478e61659c5144534/modules/ocf/manifests/extrapackages.pp#L212

Reply:
Yeah I remember trying to use that a few days ago. I think we can just add it to extrapackages? (I am happy to do so). We already have `python3.7-venv` because I puppet'd that when I added 3.7 (which reminds me I need to finish setting up the jenkins build for that).